ui/static/ui/js/listing.js

```diff
@@ -1,6 +1,78 @@
 define(['jquery', 'bootstrap', 'icheck', 'csrf'], function(jQuery) {
   'use strict';
 
+  /* global $ */
+
+  function formatGroupName(string) {
+    string = string.charAt(0).toUpperCase() + string.slice(1);
+    return string.substring(0, string.length - 1);
+  }
+
+  /* This is going to grow up to check whether
+   * the name is a valid Kerberos account
+   */
+  function isMITEmail(kerberosId) {
```

Comment:
Doesn't this really just ends need to be a check for the @ symbol? .   I think it is maybe poorly named as well.  If I pass beaver@mit.edu this will come out false, but just beaver will be true.  I think it would be better defined as `isNotEmailAddress(username)`, or `isUsername(username)` or flip the logic in the check and call it, `isEmailAddress`.